docs/secure-connections/sauce-connect/advanced/sc-kgp.md

```diff
@@ -0,0 +1,37 @@
+---
+id: sc-kgp
+title: KGP
+sidebar_label: Tunneling Protocol
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Sauce Connect Proxy&trade; Tunneling Protocol
```

Comment:
I am a bit confused about the different concepts presented here.
- The title of this doc is "KGP", meaning it will be the main heading at the top of the page, so the first paragraph should be the one that provides conceptual info about what GP is and why we care.
-  The first level2 header - "Sauce Connect Proxy&trade; Tunneling Protocol" will appear immediately under the Heading KGP, but it isn't immediately clear how these 2 concepts are connected, nor do I understand what the Sauce Connect Proxy&trade; client and server really is or whether I, as a customer need to do anything. "Client" seems to suggest I need to do something, but it is not clear what.